pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "catalystwan"
-version = "0.35.0dev2"
+version = "0.35.0dev3"
```

Comment:
please set: `export catalystwan_export_endpoints=1` when bumping version
so `pre-commit` will re-generate `ENDPOINTS.md`

Reply:
removed the bumping, since it's these changes are not that urgent to merge